README.md

```diff
@@ -13,6 +13,7 @@
    <a href="https://www.nuget.org/packages?q=Microsoft.ComponentDetection"><img alt="Nuget" src="https://img.shields.io/nuget/v/Microsoft.ComponentDetection.Common"></a>
    <a href="https://github.com/microsoft/component-detection/actions/workflows/build.yml"><img alt="GitHub Workflow Status (with event)" src="https://github.com/microsoft/component-detection/actions/workflows/build.yml/badge.svg"></a>
    <a href="https://github.com/microsoft/component-detection/actions/workflows/codeql-analysis.yml"><img alt="GitHub CodeQL Status" src="https://github.com/microsoft/component-detection/actions/workflows/codeql-analysis.yml/badge.svg"></a>
+   <a href="https://codecov.io/gh/microsoft/component-detection"><img src="https://codecov.io/gh/microsoft/component-detection/graph/badge.svg?token=2O1VQMI7EW"/></a>
```

Comment:
Why are we including the token param here? I can access with https://[codecov.io/gh/microsoft/component-detection/graph/badge.svg](https://codecov.io/gh/microsoft/component-detection/graph/badge.svg) just fine

Reply:
no idea. I just copied it from the code cov website. maybe ratelimiting?